apps/sertra/components/mod-selector/search.tsx

```diff
@@ -16,18 +16,19 @@ export const SearchBox = <T,>({
   keyExtractor,
   options,
   onSelect,
+  searchVars,
 }: SearchBoxProps<T>) => {
   const [isFocused, setFocused] = useState<boolean>(false);
 
   const [filteredOptions, setfilteredOptions] = useState(options);
   const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => {
     const results = options.filter((option) => {
```

Comment:
Maybe do something like `const searchValue = e.target.value.toLowerCase()` outside the `.filter()`, which is practically a `for`-loop. The operation is is lightweight, so maybe this doesn't offer any optimization benefits, but it still feels the right way to do it.